Package.swift

```diff
@@ -119,8 +119,7 @@ let package = Package(
             ],
             exclude: [
                 "Components/ComponentBuilder.swift.gyb"
-            ],
-            swiftSettings: [.unsafeFlags(["-Xfrontend", "-enable-experimental-concurrency"], nil)]
```

Comment:
Are those still required to be passed manually now?

Reply:
No, you don't have to manually pass in the flags to the Swift compiler. All the functionality hidden behind the flags is now be part of the toolchain without the need for any feature flags.